HW_3/model.py

```diff
@@ -0,0 +1,11 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(80), nullable=False)
+    user_surname = db.Column(db.String(80), nullable=False)
+    email = db.Column(db.String(80), unique=True, nullable=False)
+    password = db.Column(db.String(80), nullable=False)
```

Comment:
Если хэшируете при помощи `sha256`, то длина всегда будет всегда 64 символа
```suggestion
    password = db.Column(db.String(64), nullable=False)
```